File: src/easy/num_of_unplaced_fruits.rs
pub fn num_of_unplaced_fruits(fruits: Vec<i32>, baskets: Vec<i32>) -> i32 {
    let mut count: i32 = baskets.len() as i32;
    let mut replaced = vec![false; baskets.len()];
    for fruit in fruits.iter() {
        for (i, basket) in baskets.iter().enumerate() {
            if !replaced[i] && basket >= fruit {
                replaced[i] = true;
                count -= 1;
                break;
            }
        }
    }
    count
}

File: lib/easy/num_of_unplaced_fruits.dart
int numOfUnplacedFruits(List<int> fruits, List<int> baskets) {
  int count = fruits.length;
  List<bool> replaced = List.filled(baskets.length, false);
  for (final fruit in fruits) {
    for (int i = 0; i < baskets.length; i++) {
      if (baskets[i] >= fruit && !replaced[i]) {
        replaced[i] = true;
        count--;
        break;
      }
    }
  }
  return count;
}
